habitat/core/simulator.py

```diff
@@ -307,7 +324,7 @@ def reset(self) -> Observations:
         """
         raise NotImplementedError
 
-    def step(self, action: int) -> Observations:
+    def step(self, action, *args, **kwargs) -> Observations:
```

Comment:
This signature change is needed as we need a generic API here to support stepping from different simulators.